packages/zowe-explorer/src/trees/uss/USSActions.ts

```diff
@@ -82,6 +85,7 @@ export class USSActions {
                 } else {
```

Comment:
and sma equestion here... should we check for `node.fullPAth?.length === 0` here?

Reply:
We should be able to leave this as-is now that I moved up the `isTopLevel` variable 😋